Playbooks/AS-Clear-Okta-Network-Zone-List/azuredeploy.json

```diff
@@ -0,0 +1,394 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "metadata": {
```

Comment:
Hello @AcceleryntSecurityDev, 
Postdeploymentsteps are missing from metadata. Can you please add that

Reply:
@v-prasadboke this has been added